### ISSUE TITLE
Type violation via import statement does not generate error

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#3567]
Consider the following code, which should generate a type violation because `x` is assigned a value that is not compatible with the declared type.

```python
x: Callable[[], None]
from math import pow as x
```

**Comment by erictraut** [comment]
This will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.254, which I just published. It will also be included in the next release of pylance.
